src/main/kotlin/com/fasterxml/jackson/module/kotlin/ReflectionCache.kt

```diff
@@ -34,17 +32,11 @@ internal class ReflectionCache(reflectionCacheSize: Int) {
         }
     }
 
-    private val javaClassToKotlin = LRUMap<Class<Any>, KClass<Any>>(reflectionCacheSize, reflectionCacheSize)
```

Comment:
The `Class` -> `KClass` conversion is cached on the `Kotlin` side as of 1.5.32, so there is no need to cache it in `jackson-module-kotlin` anymore.
https://github.com/JetBrains/kotlin/blob/v1.5.32/core/reflection.jvm/src/kotlin/reflect/jvm/internal/kClassCache.kt